### PR TITLE
l10n: update Japanese translation

### PR DESCRIPTION
**Pull request by musashino205**
Updated Japanese translations for some new functions.

Signed-off-by: INAGAKI Hiroshi <musashino.open@gmail.com>

### CODE REVIEW

**Comment by connyduck** [comment]
thx!

File: app/src/main/res/values-ja/strings.xml [hunk 1]
```diff
@@ -199,11 +215,13 @@
 
     <string name="pref_status_text_size">トゥートのテキストサイズ</string>
 
-    <!--<string-array name="status_text_size_names">-->
-        <!--<item>小</item>-->
-        <!--<item>中</item>-->
-        <!--<item>大</item>-->
-    <!--</string-array>-->
+    <string-array name="status_text_size_names">-->
```
Comment (musashino205) [inline app/src/main/res/values-ja/strings.xml]: Sorry, I missed this ```-->```...

Comment (connyduck) [reply app/src/main/res/values-ja/strings.xml]: ah thx, I fixed it: https://github.com/tuskyapp/Tusky/commit/197badb13b8d10de789f7a60891a4bd80a9a38c1

Comment (musashino205) [reply app/src/main/res/values-ja/strings.xml]: Thank you!

**Pull request merged**